app/lib/populateCache.js

```diff
@@ -5,11 +5,12 @@ const assert = require('assert');
 
 function populateCacheSync(path) {
   const data = fs.readFileSync(path);
-  assert.ok(data.length > 0, `No data in file: '${path}'.`);
+  assert(data.length > 0, `No data in file: '${path}'.`);
 
   const records = parse(data, { columns: true });
```

Comment:
Possibly but doing that means changing this (at least how it is currently working) when the columns change. I don't want to have to make that change as this was/is intended to be a generic 'load stuff into cache'.